docker/README.md

```diff
@@ -0,0 +1,23 @@
+# Docker compose files
+
+This folder contains docker-compose files used to spin up OHIF-Viewer with differnt options such as locally or with any PAS you desire to
+
+## Public Server
+#### build
+
+`$ docker-compose -f docker-compose-publicserver.yml build`
```

Comment:
![image](https://user-images.githubusercontent.com/5797588/56742879-8eeedf80-6743-11e9-90bf-e5778eebb5d9.png)

No dice for me on `windows 10 pro`.

Reply:
I'm okay to resolve this, but we should add a note about bumping the available memory for our docker instance. You can see the `--max-old-space-size=8192` flag telling node it can hit up to ~8gb? but most instances start with ~1gb allocation.